spec/models/identification_spec.rb

```diff
@@ -1046,6 +1046,13 @@
     i.reload
     expect( i ).not_to be_disagreement
   end
+  it "should not be automatically set to true if no other identifications are current" do
+    o = make_research_grade_candidate_observation
```

Comment:
Not a major problem, but all this test requires is an observation with one withdrawn Identification, so instead of incurrent the overhead of making a research-grade candidate, you could do something like `o = Identification.make!( current: false ).observation`.